validate/validate.go

```diff
@@ -218,6 +218,23 @@ func (v *Validator) CheckProcess() (msgs []string) {
 		}
 	}
 
+	if len(process.Args) == 0 {
+		msgs = append(msgs, fmt.Sprintf("args must not be empty"))
+	} else {
+		if filepath.IsAbs(process.Args[0]) {
+			var rootfsPath string
+			if filepath.IsAbs(v.spec.Root.Path) {
+				rootfsPath = v.spec.Root.Path
+			} else {
+				rootfsPath = filepath.Join(v.bundlePath, v.spec.Root.Path)
+			}
+			absPath := filepath.Join(rootfsPath, process.Args[0])
+			if _, err := os.Stat(absPath); err != nil {
+				msgs = append(msgs, err.Error())
```

Comment:
Not finding the binary in the `root.path` is not necessarily fatal.  Perhaps a later mount or pre-start hook or some such was going to provide a binary at the configured path (previous discussion [here][1]).

[1]: https://github.com/opencontainers/runc/pull/827#r63464201